### ISSUE TITLE
A way to set Test Mode?

### ISSUE THREAD

**Issue by gpierrick** [delighted/delighted-node#15]
As suggested here https://delighted.com/docs/api/web

Is there a way to set the driver to "test"?

**Comment by mkdynamic** [comment]
@gpierrick there is a mock server that you can use for testing, scope the README for an example:
https://github.com/delighted/delighted-node#-advanced-configuration--testing

Not sure if that is exactly what you were looking for, but hopefully that helps.

**Comment by gpierrick** [comment]
Thanks. Yeah I did see it but it's exactly what I was looking for.

I was more looking to do something e.g: Braintree where you set a "sandbox" on the driver instance.

Thanks anyway!

**Comment by mkdynamic** [comment]
>I was more looking to do something e.g: Braintree where you set a "sandbox" on the driver instance.

Gotcha. That is something we don't currently provide, but it's on the roadmap. It's def. not ideal, but an alternative in the meantime is to create a separate (trial) account and use that for testing. We can configure that not to actually send anything and/or extend the trial period if it helps. Just shoot us an email at hello@delighted.com with the account email if that would help.

**Comment by gpierrick** [comment]
Yeah it would be ideal and also to remove the Throttling on the trial account so we can send it multiple times for different use cases between Development and staging.

Thanks
